### ISSUE TITLE
topk_words in Tokenizer is not sorted.

### ISSUE THREAD

**Issue by zhujiem** [xue-pai/FuxiCTR#2]
words = words[0:self._topk_words]  should be sorted.

**Comment by zhujiem** [comment]
Fixed https://github.com/xue-pai/FuxiCTR/blob/main/fuxictr/preprocess.py#L49

**Comment by zhujiem** [comment]
FM_criteo_x4_tuner_config_02

base_config: ../config/
base_expid: FM_base
dataset_id: criteo_x4

dataset_config:
    criteo_x4:
        data_root: ../data/Criteo/
        data_format: csv
        train_data: ../data/Criteo/Criteo_x4/train.csv
        valid_data: ../data/Criteo/Criteo_x4/valid.csv
        test_data: ../data/Criteo/Criteo_x4/test.csv
        min_categr_count: 10
        feature_cols:
            - {name: [I1,I2,I3,I4,I5,I6,I7,I8,I9,I10,I11,I12,I13],
               active: True, dtype: float, type: categorical, preprocess: convert_to_bucket, na_value: 0}
            - {name: [C1,C2,C3,C4,C5,C6,C7,C8,C9,C10,C11,C12,C13,C14,C15,C16,C17,C18,C19,C20,C21,C22,C23,C24,C25,C26],
               active: True, dtype: str, type: categorical, na_value: ""}
        label_col: {name: Label, dtype: float}

tuner_space:
    model_root: './Criteo/FM_criteo/min10/'
    embedding_dim: 16
    regularizer: [0, 1.e-6, 1.e-5, 1.e-4]
    learning_rate: 1.e-3
    batch_size: 10000
    seed: 2019

 20210831-232555,[command] python benchmark.py --version pytorch --config Criteo/FM_criteo_x4_001/FM_criteo_x4_tuner_config_02 --expid FM_criteo_x4_001_df584306 --gpu 0,[exp_id] FM_criteo_x4_001_df584306,[dataset_id] criteo_x4_9ea3bdfc,[train] N.A.,[val] logloss: 0.449649 - AUC: 0.803118,[test] logloss: 0.449396 - AUC: 0.803503
 20210901-024437,[command] python benchmark.py --version pytorch --config Criteo/FM_criteo_x4_001/FM_criteo_x4_tuner_config_02 --expid FM_criteo_x4_002_4661e593 --gpu 0,[exp_id] FM_criteo_x4_002_4661e593,[dataset_id] criteo_x4_9ea3bdfc,[train] N.A.,[val] logloss: 0.444749 - AUC: 0.807179,[test] logloss: 0.444486 - AUC: 0.807512
 20210901-115913,[command] python benchmark.py --version pytorch --config Criteo/FM_criteo_x4_001/FM_criteo_x4_tuner_config_02 --expid FM_criteo_x4_003_3da0082a --gpu 0,[exp_id] FM_criteo_x4_003_3da0082a,[dataset_id] criteo_x4_9ea3bdfc,[train] N.A.,[val] logloss: 0.443421 - AUC: 0.808198,[test] logloss: 0.443109 - AUC: 0.808607
 20210902-091353,[command] python benchmark.py --version pytorch --config Criteo/FM_criteo_x4_001/FM_criteo_x4_tuner_config_02 --expid FM_criteo_x4_004_3402a9bc --gpu 0,[exp_id] FM_criteo_x4_004_3402a9bc,[dataset_id] criteo_x4_9ea3bdfc,[train] N.A.,[val] logloss: 0.449190 - AUC: 0.801985,[test] logloss: 0.448863 - AUC: 0.802439
